### ISSUE TITLE
Parser regression in 4.4.25

### ISSUE THREAD

**Issue by olafurpg** [scalameta/scalameta#2442]
To reproduce: clone the zio/zio repository and run the following commands
```sh
❯ cs launch org.scalameta:scalafmt-cli_2.13:2.7.5 org.scalameta:scalameta_2.13:4.4.25 -- --stdout benchmarks/src/main/scala/zio/QueueBackPressureBenchmark.scala  | head
/Users/olafurpg/dev/zio/zio/benchmarks/src/main/scala/zio/QueueBackPressureBenchmark.scala:21: error: expected start of definition
 */
   ^
❯ cs launch org.scalameta:scalafmt-cli_2.13:2.7.5 org.scalameta:scalameta_2.13:4.4.24 -- --stdout benchmarks/src/main/scala/zio/QueueBackPressureBenchmark.scala  | head
package zio

import cats.effect.{ContextShift, IO => CIO}
import monix.eval.{Task => MTask}
import org.openjdk.jmh.annotations._
import zio.IOBenchmarks._
import zio.stm._

import java.util.concurrent.TimeUnit
import scala.concurrent.ExecutionContext
```
The parse error is only present with Scalameta 4.4.25 on the classpath, not 4.424. The docstring between annotations and the class definition seems to be causing the issue
https://github.com/zio/zio/blob/9e11682722a357e8ae8789f3fffa5f02c31637f3/benchmarks/src/main/scala/zio/QueueBackPressureBenchmark.scala#L18-L20

**Comment by dos65** [comment]
This problem with comments should be fixed by #2437

**Comment by dos65** [comment]
Yep, this one is fixed. Checked it by running community-test against zio repo.
